### PR TITLE
add_velocity function

### PR DESCRIPTION
**Pull request by PalkaPuri**

### CODE REVIEW

File: collab2/foraging/toolkit/velocity.py [hunk 1]
```diff
@@ -3,93 +3,20 @@
...
-
-    velocity_scoresDF = pd.concat(velocity_scores)
-
-    return {"velocity_scores": velocity_scores, "velocity_scoresDF": velocity_scoresDF}
+# a function that calculates the velocity magnitude and direction and outputs updated (foragers, foragersDF)
+def add_velocity(foragers: List[pd.DataFrame], dt: int = 1):
+    for df in foragers:
+        v_ID = f"v_dt={dt}"
+        theta_ID = f"theta_dt={dt}"
+        if v_ID in df.columns and theta_ID in df.columns:
```
Comment (rfl-urbaniak) [inline collab2/foraging/toolkit/velocity.py:12]: I the columns already existed, the user should be aware of this, instead of silent behavior.

**Review by rfl-urbaniak — CHANGES_REQUESTED** [review]
Only now I noticed that foragers get assigned floats instead of integers. Can you please fix this in the same PR?

**Comment by PalkaPuri** [comment]
> Only now I noticed that foragers get assigned floats instead of integers. Can you please fix this in the same PR?

Fixed by adding a line in dataObject

File: collab2/foraging/toolkit/velocity.py (hunk 1 above)
Comment (PalkaPuri) [reply collab2/foraging/toolkit/velocity.py:12]: Agreed. Added warning

**Review by rfl-urbaniak — APPROVED** [review]

**Pull request merged**